jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/test/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImplTest.java

```diff
@@ -55,4 +91,60 @@ private void assertMethodResult(final Method method, final Class<?> returnType,
         }
     }
 
+    @Test
+    public void testGetProcesses(){
```

Comment:
It looks like RemoteProcessRuntimeDataServiceImpl class didn't have decent coverage when it comes to tests in the past.
Now you added tests which cover getting processes. Should we plan to expand the coverage to other methods in the near future or there are more changes planned in this area?